api/descriptions/users/id_delete.md

```diff
@@ -1,11 +1,24 @@
-Deletes a user from the system by username.
+Deletes a user from the system.
```

Comment:
@KICKdesigns Let's extend our template to always document URL parameters in some sort of standard way. I think almost all of our URL params are called `{id}`. We've got two shortcomings today:

* Our spec file doesn't have a description for URL params.
* Since the name of the URL param is always `id`, it's not helpful in figuring out what value to plug into it.